kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -608,7 +627,11 @@ func (options *SubmitJobOptions) Run(ctx context.Context, factory cmdutil.Factor
 }
 
 func (options *SubmitJobOptions) raySubmitCmd() ([]string, error) {
-	raySubmitCmd := []string{"ray", "job", "submit", "--address", dashboardAddr}
+	addr := options.address
+	if addr == "" {
+		addr = dashboardAddr
+	}
+	raySubmitCmd := []string{"ray", "job", "submit", "--address", addr}
 
 	if len(options.runtimeEnv) > 0 {
```

Comment:
Unrelated: would be great if we can figure out a better way than hardcoding for each CLI flag.